docs/install/iso-install.md

```diff
@@ -87,17 +87,31 @@ The following [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of an
 	For more information, see [DHCP Server Configuration](./pxe-boot-install.md#dhcp-server-configuration).
 	:::
 
+1. (Optional) Configure the CIDRs for the cluster pods and services. Leave these fields blank to use the default CIDRs.
+
+	![config-cluster-cidrs.png](/img/v1.5/install/config-cluster-cidrs.png)
+
+  The following is an example of a valid CIDR configuration:
+
+  * Pod CIDR: 172.16.0.0/16
+  * Service CIDR: 172.22.0.0/16
+  * Cluster DNS IP: 172.22.0.10
+
+	:::note
+  The pod CIDR and service CIDR must not overlap. The DNS service IP must be within the service CIDR.
+	:::
+
 1. (Optional) Configure the `DNS Servers`. Use commas as a delimiter to add more DNS servers. Leave it blank to use the default DNS server.
 
 	![config-dns-server.png](/img/v1.2/install/config-dns-server.png)
 
 1. Configure the virtual IP (VIP) by selecting a `VIP Mode`. This VIP is used to access the cluster or for other nodes to join the cluster.
 
 	:::note
-	If using DHCP to configure the IP address, you need to configure a static MAC-to-IP address mapping on your DHCP server to have a persistent virtual IP (VIP), and the VIP must be unique.
+	For DHCP setup with static MAC-to-IP address mappings configured, enter the MAC address in the provided field to fetch the unique persistent virtual IP (VIP). Otherwise, leave it blank.
```

Comment:
A question about this statement:

In DHCP mode, even if server side has no pre-configured IP-MAC binding, can the user still enter a preferred MAC? Namely, user is free to enter any valid MAC.

Reply:
Good point. If user provides a non-configured MAC, DHCP will just issue a new IP. The `vip-*@mgmt-br` links end up with that particular IP-MAC pairing.

Taking my testing a bit further, if the installer sends the MAC address of another cluster's VIP, DHCP will blindly return the same VIP of that particular cluster to the installer. If the user is unaware that IP is already assigned, it can ruin the installation. At the moment, I don't know if there is an option we can send to DHCP to for example, error out if the IP is already in-use. I believe the same problem also exists in the current automatic installation modes.

Reply:
The latter is a networking plan/management problem, not a Harvester issue. Luckily, most business customers have processes to make sure the IP/MAC... are well managed.

Reply:
Fixed in 99a2455f32ad0b21de0e09788fa12529a3551fc1.